### DISCUSSION TITLE
Up the Bus Factor for releases?

### DISCUSSION THREAD

**Discussion by carltongibson** [django-compressor/django-compressor#1259] [General]
Hi @django-compressor/core 👋 

It's summer, and I think @diox is on holiday. (I live in Europe, so I think that kind of matters). 

At the same time we need to merge and release https://github.com/django-compressor/django-compressor/pull/1257 — for Django 5.1 compat. 

I don't have PyPI access for django-compressor, so I can't just roll it myself.

It would be good to just up the bus factor a little here. WDYT? Thanks.

**Comment by diox** [comment]
Hi,

I do agree about needing to up the bus factor - there used to be a couple more people with access but they are no longer active (and I removed them from the PyPI team as they did not have 2FA set up anyway).

Give me your PyPI handle and I'll add you.

**Reply by carltongibson** [reply]
I'm @carltongibson there too. 

THIS IS ME: https://pypi.org/user/carltongibson/

**Reply by diox** [reply]
Sent an invite, it's pending

**Reply by carltongibson** [reply]
Accepted. I'll roll 4.5.1 tomorrow. 🥳
